pkg/kubefed2/federate/disable.go

```diff
@@ -121,9 +122,21 @@ func (j *disableType) Run(cmdOut io.Writer, config util.FedConfig) error {
 		return errors.Wrap(err, "Failed to get host cluster config")
 	}
 
+	// If . is specified, the target name is assumed as a group qualified name.
+	// In such case, ignore the lookup to make sure deletion of a federatedtypeconfig
+	// for which the corresponding target has been removed.
```

Comment:
just curious: in what case is this extra check necessary?

Reply:
This command can't assume that the target crd exists at the time it is invoked (in which case lookup would fail), and I think it's nicer to delete/disable the ftc rather than failing if that occurs.